bugbot/crash/analyzer.py

```diff
@@ -477,6 +477,41 @@ def is_near_allocator_related_crash(self) -> bool:
         """
         return self.is_near_allocator_crash or self.is_potential_near_allocator_crash
 
+    @property
+    def is_content_crash(self) -> bool:
+        """Whether the crash is related to content process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "content":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_gpu_crash(self) -> bool:
+        """Whether the crash is related to GPU."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "gpu":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_parent_crash(self) -> bool:
+        """Whether the crash is related to the parent process."""
+        for row in self.signature["facets"]["process_type"]:
```

Comment:
I'm not familiar with the structure of whatever thing you are iterating over. What does it mean for them to have multiple things in the process_type? Or does it usually just have one and you are iterating over them for completeness?